SU2_CFD/src/output/CFlowOutput.cpp

```diff
@@ -2724,6 +2732,14 @@ void CFlowOutput::WriteForcesBreakdown(const CConfig* config, const CSolver* flo
           file << "Laminar Viscosity (non-dim): " << config->GetMu_ConstantND() << "\n";
           break;
 
+        case VISCOSITYMODEL::COOLPROP:
+          file << "Viscosity Model: CoolProp \n";
+          file << "Constant Laminar Viscosity: " << config->GetMu_Constant();
+          if (si_units) file << " N.s/m^2.\n";
+          else file << " lbf.s/ft^2.\n";
+          file << "Laminar Viscosity (non-dim): " << config->GetMu_ConstantND() << "\n";
```

Comment:
My comment on reporting a constat value also applies to all these other places, of course.

Reply:
haha, you are right.
I did this for all output files, including viscosity and conductivity.